### ISSUE TITLE
Feedback from Morgan and Diane

### ISSUE THREAD

**Issue by msschwartz21** [dlmbl/intro_dl#3]
- [ ] Add colored markdown blocks for tasks
- [ ] Add exercise checkpoints
- [ ] Please give students the code for plotting the perceptron in task 1
- [ ] Task 3.1 (expand on a simple tensorflow model) is very open ended. Can we give students more structure? We like the idea of structuring the task like an ablation study, e.g. look at the impact of changing each component of the model (activation, number of layers, loss, etc)
- [ ] Please give students the plotting code for task 3.2 or provide more scaffolding so that the task is more approachable.
- [ ] Would it be possible to switch to a biological dataset if we do all the data wrangling for the students? The Broad collection has some potential options (https://bbbc.broadinstitute.org/image_sets). 
- [ ] Task 4.1: please give the students the appropriate function names to use for each layer, e.g. `Conv2D` and provide a link to relevant Keras docs
- [ ] Task 4.2: could benefit from more hints than just "see boot"
- [ ] Convert task 4.3 to a bonus exercise
